src/test_l1_plat_power.c

```diff
@@ -74,6 +74,30 @@
 static int gTestGroup = 1;
 static int gTestID = 1;
 
+#define PLAT_ASSERT_AUTO_TERM_NUMERICAL(value, comparison){\
+      if(value != comparison){\
+	        UT_LOG("\n In %s Comparison: [%d = %d]\n", __FUNCTION__, value, comparison);\
+		PLAT_TERM();\
+		UT_FAIL_FATAL();\
+      }\
+}\
+
+#define PLAT_ASSERT_AUTO_TERM_NOT_EQUAL(value, comparison){\
```

Comment:
PLAT_ASSERT_AUTO_TERM_NOT_EQUAL -> UT_ASSERT_FATAL( value != comparision )

Reply:
Proper clean up should be done before exit.